server/cmd/mattermost/commands/import.go

```diff
@@ -57,6 +57,8 @@ func slackImportCmdF(command *cobra.Command, args []string) error {
 	}
 	defer a.Srv().Shutdown()
 
+	var rctx = request.EmptyContext(a.Log())
```

Comment:
Nit pick: It's idiomatic in go to use `:=` when possible.
```suggestion
	rctx := request.EmptyContext(a.Log())
```

Reply:
Okay I get it :+1: